<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-4L006RJ" ignored="1" name="Test1" tests="8" failures="1" timestamp="10 May 2023 10:07:51 GMT" time="1.151" errors="0">
  <testcase name="M1" time="0.009" classname="testNgExamples2may.Withouttestngclass"/>
  <testcase name="M1" time="0.002" classname="testNgExamples2may.Withouttestngclass"/>
  <testcase name="M1" time="0.003" classname="testNgExamples2may.Withouttestngclass"/>
  <testcase name="LogIn" time="0.002" classname="testNgExamples2may.Dependsonmetod"/>
  <testcase name="LogOut" time="1.043" classname="testNgExamples2may.Dependsonmetod">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method testNgExamples2may.Dependsonmetod.LogOut() didn&amp;apos;t finish within the time-out 1000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method testNgExamples2may.Dependsonmetod.LogOut() didn't finish within the time-out 1000
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 2 stack frames]]>
    </failure>
  </testcase> <!-- LogOut -->
  <testcase name="tc2" time="0.002" classname="testNgExamples2may.simpleprogram"/>
  <testcase name="SignIn" time="0.002" classname="testNgExamples2may.Dependsonmetod"/>
  <testcase name="tc1" time="0.002" classname="testNgExamples2may.simpleprogram"/>
  <testcase name="tc4" time="0.0" classname="testNgExamples2may.simpleprogram">
    <ignored/>
  </testcase> <!-- tc4 -->
</testsuite> <!-- Test1 -->
